src/components/RNMarkdownTextInput.tsx

```diff
@@ -27,6 +28,11 @@ function RNMarkdownTextInputWithRef(props: MarkdownTextInputProps, ref: Forwarde
             }}
             // eslint-disable-next-line
             {...props}
+            /**
+             * If maxLength is not set, we should set the it to CONST.MAX_COMMENT_LENGTH + 1, to avoid parsing markdown for large text
+             * Since we want the ExceededCommentLength to be displayed, we need to set the maxLength to CONST.MAX_COMMENT_LENGTH + 1
```

Comment:
The second sentence is pretty similar and I don't understand the `ExceededCommentLength` bit. Is it possible to merge the two sentences, or am I missing why we need both?